features/supplier/supplier_applies_for_an_opportunity.feature

```diff
@@ -332,9 +332,9 @@ Scenario: Supplier changes their answers before submission
 
   When I click the 'View your account' link
   And I click the 'View your opportunities' link
```

Comment:
Bring in line with briefs changes from yday:
https://github.com/alphagov/digitalmarketplace-briefs-frontend/pull/63